vercel.json

```diff
@@ -740,6 +740,26 @@
             "source": "/docs/data/user-properties",
             "destination": "/docs/product-analytics/user-properties"
         },
+        {
+            "source": "/docs/feature-flags/multivariate-flags",
+            "destination": "/docs/feature-flags/creating-feature-flag"
+        },
+        {
+            "source": "/docs/feature-flags/libraries",
+            "destination": "/docs/feature-flags/"
+        },
+        {
+            "source": "/docs/feature-flags/manual",
+            "destination": "/docs/feature-flags/installation"
+        },
+        {
+            "source": "/docs/feature-flags/rollout-strategies",
+            "destination": "/docs/feature-flags/creating-feature-flags#release-conditions"
```

Comment:
```suggestion
            "destination": "/docs/feature-flags/creating-feature-flags"
```

Don't create redirects to jump links. If we change the heading, they'll create a 404 and they're not "real" URLs.

Reply:
Didnt know this! Assumed it would just redirect to the main page. Thanks!